app/kumactl/cmd/install/install_transparent_proxy.go

```diff
@@ -297,18 +283,7 @@ func configureTransparentProxy(cmd *cobra.Command, args *transparentProxyArgs) e
 		MaxRetries:                args.MaxRetries,
 		SleepBetweenRetries:       args.SleepBetweenRetries,
 	}
-
-	if args.UseTransparentProxyEngineV1 {
-		tp = transparentproxy.V1()
-
-		// best effort cleanup before we apply the rules (again?)
-		_, err := tp.Cleanup(cfg)
-		if err != nil {
-			return errors.Wrapf(err, "unable to invoke cleanup")
-		}
-	} else {
-		tp = transparentproxy.V2()
-	}
+	tp = transparentproxy.V2()
```

Comment:
Nit: There is no V1 now, so maybe we should also change the name from `V2` to something else?